package/com.unity.formats.usd/Tests/Editor/EditorTests.cs

```diff
@@ -9,5 +12,35 @@ public void DummyTest()
         {
         }
     }
+    
+    public class ScopeLoadingTest
+    {
+        private GameObject usdRoot;
+        private string filepath = "test_#154/test_collections.usda";
+        
+        [SetUp]
+        public void SetUp()
+        {
+            InitUsd.Initialize();
+            var stage = pxr.UsdStage.Open(Path.Combine(Application.dataPath, filepath), pxr.UsdStage.InitialLoadSet.LoadNone);
+            var scene = Scene.Open(stage);
+            usdRoot = USD.UsdMenu.ImportSceneAsGameObject(scene);
+            scene.Close();
+        }
+        
+        [Test]
+        public void TestScopeWithoutChildren()
```

Comment:
We should apply some rules on the test naming. You would like to be able what broke from a log without to have to open the code. My rule of thumb: `<LogicName/MethodName>_<Input>_<ExpectedOutput>`.

Names become long but this is necessary to prevent time consuming back and forth between Yamato and a local repository.